### ISSUE TITLE
ESLint error importing module

### ISSUE THREAD

**Issue by hhg2288** [maisano/react-router-transition#61]
when I try to import the library, I get eslint errors:
```
import { AnimatedSwitch } from 'react-router-transition';
```

the errors are:
```
[eslint] Unable to resolve path to module 'react-router-transition'. (import/no-unresolved)
[eslint] Missing file extension for "react-router-transition" (import/extensions)
```

I would love to know whats the error, so I can myself fix this with a PR. I want to contribute!

**Comment by monkeySmoke** [comment]
I meet the same question

**Comment by observerslee** [comment]
It looks like the package does not contain an entry point at the root directory.

<img width="215" alt="screen shot 2017-09-08 at 12 50 43 pm" src="https://user-images.githubusercontent.com/2031821/30196604-8e114fae-9494-11e7-8d9d-4367c7fcbc3b.png">

Temporarily make the error goes away by importing 

```javascript
'import { spring, AnimatedRoute } from 'react-router-transition/src';
```

**Comment by maisano** [comment]
might have something to do with the package.json–i forgot to update the `modules` and `jsnext:main` fields. that said, this very likely depends on your webpack/eslint config.

**Comment by hhg2288** [comment]
not quite sure what to do... review my eslint config? or check updates on [this](https://github.com/maisano/react-router-transition/issues/62) bug?

is there something I can do to help?

**Comment by maisano** [comment]
@hhg2288 just published `1.0.1`–let me know if this resolves the issue?

**Comment by hhg2288** [comment]
now its working!

the only thing I needed to add additionally was the Babel plugin: [`transform-object-rest-spread`](https://babeljs.io/docs/plugins/transform-object-rest-spread/)

thanks!
